src/components/NavBar.js

```diff
@@ -33,13 +34,15 @@ const NavBar = () => {
     }
   
   return (
-          <nav className="w-full bg-gray-100 shadow">
+          <nav className="w-full bg-gray-200 shadow">
             <div className="justify-between px-4 mx-auto lg:max-w-7xl md:items-center md:flex md:px-8">
               {/* Logo and menu button */}
                 <div>
                     <div className="flex items-center justify-between py-3 md:py-5 md:block">
                         <a href="/">
-                            <h2 className="text-2xl font-bold text-black">DEMO</h2>
+                            {/* <h2 className="text-2xl font-bold text-black">DEMO</h2> */}
```

Comment:
If you're going to remove code, then I would delete it entirely. Commenting out old stuff and leaving that in will end up with code littered with random comments no one knows what to do with

Reply:
I like the idea of having a logo to click on, is there a way to make it change color when the user hovers their mouse over it to give more indication that it's a clickable action?

Reply:
Yeah, my original idea was to create the branch so you guys could view it and make a decision (likes/dislikes).  So I wanted to leave the original stuff in there, but noted.  No problem.

Reply:
Sweet 👍